client/src/views/Dashboard/Dashboard.js

```diff
@@ -35,35 +40,87 @@ const Dashboard = props => {
     'Due Date',
     'Status'
   ]
+
+  const SelfReviewListingHeader = [
+    'Projects',
+    'From date',
+    'To date',
+    'Due Fom Date',
+    'Status'
+  ]
   const peerReviews = useSelector(
     state => state.peerReviewReducer.userPeerReview
   )
+  const userSelfReviews = useSelector(
+    state => state.selfReviewReducer.userSelfReviewDeatils
+  )
+  const projectData = useSelector(state => state.projectReducer.projects)
+  // const employeeData = useSelector(state => state.EmployeeInfo.employeeData)
 
   useEffect(() => {
-    dispatch(loadAllPeerForUser())
+    dispatch(loadAllPeerForUser(currentUser._id))
+    dispatch(loadAllSelfReviewsForUser(currentUser.employee_id))
   }, [dispatch])
 
+  useEffect(() => {
+    if (userSelfReviews && userSelfReviews.length > 0) {
+      // dispatch(loadAllEmployeeData())
+      dispatch(loadAllProjects())
+    }
+  }, [dispatch, userSelfReviews])
+
   const tempArr = []
   let filteredEmployee
   if (peerReviews) {
-    filteredEmployee = peerReviews.data.data.filter(
+    filteredEmployee = peerReviews.filter(
       cls =>
-        cls.employee_under_review
-          .toLowerCase()
-          .includes(searchText.toLowerCase().trim()) &&
+        cls.employee_under_review.firstname +
+          ' ' +
+          cls.employee_under_review.lastname
+            .toLowerCase()
+            .includes(searchText.toLowerCase().trim()) &&
         cls.status !== 'Done' &&
```

Comment:
is this filtering possible at server side?

Reply:
Will update api in separate branch

Reply:
ok